src/utils/noticon2gridicon.js

```diff
@@ -16,6 +16,7 @@ export const noticon2gridicon = c =>
       '\uf414': 'warning',
       '\uf418': 'checkmark',
       '\uf447': 'cart',
+      'offline': 'offline'
```

Comment:
this is a bit unsettling. did we check with the mobile teams before sending notifications with a non-noticon point reference?

Reply:
Indeed, this was one of today's tasks :) It looks like the mobile apps (at least iOS which I checked) uses a Noticon font, so indeed this may be problematic; posted for some guidance in p77Llu-a5H-p2.

Reply:
I hardcoded the offline icon in XCode today to see what would happen, and it was basically what we expected: a broken icon, because it's not a recognizable character in the Noticon font:

<img width="488" alt="screen shot 2018-04-12 at 11 14 21 am" src="https://user-images.githubusercontent.com/349751/38696138-1c91884a-3e43-11e8-999a-e62603cfb890.png">